commit 23b75093d5599baeb816bbf0163ade78d2201818
Author: Michele Calgaro <michele.calgaro@yahoo.it>
Date:   Sat Nov 22 20:39:15 2025 +0900

    twin: fix tiling of maximized windows. This resolves issue #675
    
    Signed-off-by: Michele Calgaro <michele.calgaro@yahoo.it>
    (cherry picked from commit d2b2363d9345e74035e0585344d7150c4a62ef1c)

diff --git a/twin/geometry.cpp b/twin/geometry.cpp
index 88d0ba8cd..bdef2356c 100644
--- a/twin/geometry.cpp
+++ b/twin/geometry.cpp
@@ -1915,12 +1915,12 @@ void Client::changeMaximize( bool vertical, bool horizontal, bool adjust )
     TQRect clientArea = workspace()->clientArea( MaximizeArea, this );
 
     // save sizes for restoring, if maximalizing
-    if( !activeTiled && !adjust && !( y() == clientArea.top() && height() == clientArea.height()))
+    if( !adjust && !( y() == clientArea.top() && height() == clientArea.height()))
         {
         geom_restore.setTop( y());
         geom_restore.setHeight( height());
         }
-    if( !activeTiled && !adjust && !( x() == clientArea.left() && width() == clientArea.width()))
+    if( !adjust && !( x() == clientArea.left() && width() == clientArea.width()))
         {
         geom_restore.setLeft( x());
         geom_restore.setWidth( width());
